Description: run python-config directly
 This patch runs /usr/bin/python-config directly for python 3.3
Author: Sebastien Bacher <seb128@ubuntu.com>
Origin: http://launchpadlibrarian.net/125114732/py3cairo_1.10.0%2Bdfsg-3~exp3_1.10.0%2Bdfsg-3~exp3ubuntu1.diff.gz
Last-Update: 2013-03-30

--- py3cairo-1.10.0+dfsg.orig/waflib/Tools/python.py
+++ py3cairo-1.10.0+dfsg/waflib/Tools/python.py
@@ -169,7 +169,7 @@ def check_python_headers(conf):
 		conf.find_program('python-config-%s'%num,var='PYTHON_CONFIG',mandatory=False)
 	includes=[]
 	if conf.env.PYTHON_CONFIG:
-		for incstr in conf.cmd_and_log(conf.env.PYTHON+[conf.env.PYTHON_CONFIG,'--includes']).strip().split():
+		for incstr in conf.cmd_and_log([conf.env.PYTHON_CONFIG,'--includes']).strip().split():
 			if(incstr.startswith('-I')or incstr.startswith('/I')):
 				incstr=incstr[2:]
 			if incstr not in includes:
